formats/publication/publisher/links.json

```diff
@@ -0,0 +1,28 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "additionalProperties": false,
+  "required": [
+    "lead_organisations"
+  ],
+  "properties": {
+    "document_collections": {
```

Comment:
I'm continuing to match other formats, eg Case Studies: https://github.com/alphagov/govuk-content-schemas/blob/master/formats/case_study/publisher/links.json#L24-L26

Presumably the document collection work needs a PR to remove that block from the detailed guides and case studies links hash?